code/modules/power/generator.dm

```diff
@@ -113,12 +124,29 @@
 		circ2.network2.update = 1
 
 	//Exceeding maximum power leads to some power loss
-	if(effective_gen > max_power && prob(5))
-		var/datum/effect/effect/system/spark_spread/s = new /datum/effect/effect/system/spark_spread
-		s.set_up(2, 1, src)
-		s.start()
-		stored_energy *= 0.5
-
+	//exceeding max power is supposed to be bad, have there be a small chance of some visual and audio effects to stress this
+	//remember to lubricate your engines kiddos
+	if(!lubricated)
+		if(effective_gen > max_power && prob(5))
+			var/datum/effect/effect/system/spark_spread/s = new()
+			s.set_up(2, 1, src)
+			s.start()
+			stored_energy *= 0.5
+			if(prob(55))
+				visible_message("<span class='danger'>[src] [pick(soundverb)]!</span>")
+				var/malfsound = pick(soundlist)
+				playsound(src.loc, malfsound, 50, 0, 10)
+				if(prob(20))
+					var/datum/effect/effect/system/smoke_spread/SM = new()
+					SM.set_up(5, 0, src.loc)
+					playsound(src.loc, 'sound/machines/warning-buzzer.ogg', 50, 1, -3)
+					spawn(2 SECONDS)
+						playsound(src.loc, 'sound/effects/meteorimpact.ogg', 50, 1, -3)
+						for(var/mob/living/M in view(7, src))
+							shake_camera(M, 1, 2)
+						spawn(5)
```

Comment:
0.5 SECONDS works just as well. I'm eternally averse to spawn after hearing about its performance, but given that this is nested enough to be classified as a bird's nest I'm sure it'll be fine.

Reply:
Where do you think the Teshari come from?